features/include/pcl/features/intensity_gradient.h

```diff
@@ -107,8 +117,8 @@ namespace pcl
     protected:
       ///intensity field accessor structure
       IntensitySelectorT intensity_;
-      ///number of threads to be used, default 0 (auto)
-      unsigned int threads_{0};
+      ///number of threads to be used
+      unsigned int num_threads_{1};
```

Comment:
This defaulted to 0, ie. automatic number of threads, but now it defaults to 1 (as majority of the algorithms did) - but I guess we don't have any rule here, whether or not to include it?